tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -52,3 +57,29 @@ def dismiss_tooltip(self):
         confirm_element = Wait(self.marionette).until(expected.element_present(*self._button_popup_intro))
         Wait(self.marionette).until(expected.element_displayed(confirm_element))
         confirm_element.tap()
+
+class BugzillaLiteStage(BugzillaLite):
+
+    name =  'Bugzilla Lite Stage'
+    manifest_url = 'http://bzlite-staging.herokuapp.com/manifest.webapp'
+
+    def launch(self):
+        Base.launch(self)
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._popup_intro))))
+
+    def dismiss_content(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._content_locator))))
+        close_element = self.marionette.find_element(*self._close_locator)
```

Comment:
You might as well do self.marionette.find_element(*self._close_locator).tap() here.